backend/src/tests/searchTerm.service.test.ts

```diff
@@ -0,0 +1,45 @@
+import { SearchTermService } from '../services/searchTerm.service';
+
+describe('SearchTermService', () => {
+  const searchTermService = new SearchTermService();
+
+  test('findSearchItem should return limited suggestions', async () => {
+    const searchTerm = {
+      searchTerm: 'test',
+      limit: 2,
+    };
+    const results = await searchTermService.findSearchItem(searchTerm);
```

Comment:
Great job on the tests. Bit of good practice nitpicking, try to put a space between the different logic blocks:
```
// Define/Prepare

// Expression/Act

// Assert/Expectation
```